Automattic/vip-go-mu-plugins
# VIP Cache Manager: implement "Purge page and its assets" admin bar button

## Description

This PR adds an Admin Bar button that allows a user to purge the current page and all its external (as opposed to inline) styles and scripts.

<img width="253" alt="Screen Shot 2020-11-23 at 8 24 01 PM" src="https://user-images.githubusercontent.com/459254/100039534-06c33700-2dcb-11eb-9c73-83bdd1e3818e.png">

<img width="150" alt="Screen Shot 2020-11-23 at 8 24 09 PM" src="https://user-images.githubusercontent.com/459254/100039549-0aef5480-2dcb-11eb-92c6-9b8b5734a371.png">

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Make sure you're logged in.
1. Go to any frontend page and verify the button is there and working
1. Verify cookies are delicious.

## Considerations

- Do we also want to display the button in the admin to allow purging the assets for Dashboard requests?
- Do we want to add a filter to allow users to adjust the needed capability? Currently, it's hardcoded for `manage_options` which may limit the usefulness.
